docs/src/content/docs/es/challenges/angular/3-directive-enhancement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 🟠 Mejorar Directiva
+description: El desafío 3 se trata de una directive incorporada.
+author: thomas-laforge
+challengeNumber: 3
+command: angular-ngfor-enhancement
+blogLink: https://medium.com/@thomas.laforge/ngfor-enhancement-716b44656a6c
+sidebar:
+  order: 101
+---
+
+:::note
```

Comment:
Nitpick: this `Note` title I believe can also be translated. I believe the syntax is:

```
:::note[Nota]

text

:::
```

Reply:
do you have any doc site to confirm the syntax of this markup?

Reply:
Don't quite remember the docs reference but I implemented it here 

https://github.com/tomalaforge/angular-challenges/blob/main/docs/src/content/docs/es/challenges/performance/34-default-onpush.md?plain=1#L19

![image](https://github.com/tomalaforge/angular-challenges/assets/62297014/f5d9f15f-a437-4070-9188-f1757c73c8fb)

Reply:
Found it :) https://starlight.astro.build/guides/authoring-content/#custom-aside-titles

Reply:
Nice. Thanks!

Fixed.